pkg/apis/kops/cluster.go

```diff
@@ -470,6 +471,10 @@ type KubeDNSConfig struct {
 	CacheMaxSize int `json:"cacheMaxSize,omitempty"`
 	// CacheMaxConcurrent is the maximum number of concurrent queries for dnsmasq
 	CacheMaxConcurrent int `json:"cacheMaxConcurrent,omitempty"`
+	// Tolerations	are tolerations to apply to the kube-dns deployment
```

Comment:
This must be added to `pkg/apis/kops/v1alpha2/cluster.go` as well. Then you need to run `make crds apimachinery`

Reply:
I've fixed that, the GitHub workflows need a new approval before they're re-run